examples/acoustics_2d_radial/setrun_abl.py

```diff
@@ -0,0 +1,260 @@
+    # at AMR levels <= verbosity.  Set verbosity = 0 for no printing.
+    #   (E.g. verbosity == 2 means print only on levels 1 and 2.)
+    clawdata.verbosity = 0
+
+
+
+    # --------------
+    # Time stepping:
+    # --------------
+
+    # if dt_variable==True:  variable time steps used based on cfl_desired,
+    # if dt_variable==False: fixed time steps dt = dt_initial always used.
+    clawdata.dt_variable = True
+
+    # Initial time step for variable dt.
+    # (If dt_variable==0 then dt=dt_initial for all steps)
+    clawdata.dt_initial = 1.00000e-02
+
+    # Max time step to be allowed if variable dt used:
+    clawdata.dt_max = 1.000000e+99
+
+    # Desired Courant number if variable dt used
+    clawdata.cfl_desired = 0.900000
+    # max Courant number to allow without retaking step with a smaller dt:
+    clawdata.cfl_max = 1.000000
+
+    # Maximum number of time steps to allow between output times:
+    clawdata.steps_max = 50000
+
+
+    # ------------------
+    # Method to be used:
+    # ------------------
+
+    # Order of accuracy:  1 => Godunov,  2 => Lax-Wendroff plus limiters
+    clawdata.order = 2
+
+    # Use dimensional splitting? (not yet available for AMR)
+    clawdata.dimensional_split = 'unsplit'
+
+    # For unsplit method, transverse_waves can be
+    #  0 or 'none'      ==> donor cell (only normal solver used)
+    #  1 or 'increment' ==> corner transport of waves
+    #  2 or 'all'       ==> corner transport of 2nd order corrections too
+    clawdata.transverse_waves = 2
+
+
+    # Number of waves in the Riemann solution:
+    clawdata.num_waves = 2
+
+    # List of limiters to use for each wave family:
+    # Required:  len(limiter) == num_waves
+    # Some options:
+    #   0 or 'none'     ==> no limiter (Lax-Wendroff)
+    #   1 or 'minmod'   ==> minmod
+    #   2 or 'superbee' ==> superbee
+    #   3 or 'vanleer'  ==> van Leer
+    #   4 or 'mc'       ==> MC limiter
+    clawdata.limiter = ['vanleer','vanleer']
+
+    clawdata.use_fwaves = False    # True ==> use f-wave version of algorithms
+
+    # Source terms splitting:
+    #   src_split == 0 or 'none'    ==> no source term (src routine never
+    #   called)
+    #   src_split == 1 or 'godunov' ==> Godunov (1st order) splitting used,
+    #   src_split == 2 or 'strang'  ==> Strang (2nd order) splitting used,
+    #   not recommended.
+    clawdata.source_split = 0
+
+
+    # --------------------
+    # Boundary conditions:
+    # --------------------
+
+    # Number of ghost cells (usually 2)
+    clawdata.num_ghost = 2
+
+    # Choice of BCs at xlower and xupper:
+    #   0 or 'user'     => user specified (must modify bcNamr.f to use this
+    #   option)
+    #   1 or 'extrap'   => extrapolation (non-reflecting outflow)
+    #   2 or 'periodic' => periodic (must specify this at both boundaries)
+    #   3 or 'wall'     => solid wall for systems where q(2) is normal
+    #   velocity
+
+    clawdata.bc_lower[0] = 'extrap'   # at xlower
```

Comment:
Perhaps comment somewhere that with ABL it's best to set `extrap` BCs at the edges of the layer, which is what's done here.